build.gradle.kts

```diff
@@ -88,6 +89,7 @@ subprojects {
   apply(plugin = "checkstyle")
   apply(plugin = "ru.vyarus.animalsniffer")
   apply(plugin = "biz.aQute.bnd.builder")
+  apply(plugin = "io.github.usefulness.maven-sympathy")
```

Comment:
May be worth only applying to modules which are published, but if it's not too invasive as-is don't worry about it.